modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/model.scala

```diff
@@ -130,18 +130,32 @@ object Status {
   * A list of Sequences. The `Queue` could be empty of Sequences when waiting
   * for the addition of new ones.
   */
-case class Queue[A](sequences: List[Sequence[A]]) {
+case class Queue[A](
+  sequences: List[Sequence[A]],
+  // Unrooted Steps (without a Queue)
```

Comment:
Somehow the "unrooted" state is very unclear to me. Why would there be steps not in the queue? And the same for executions